src/apps/explorer/pages/Home/index.tsx

```diff
@@ -12,22 +13,27 @@ const Wrapper = styled.div`
   width: 100%;
 
   > h1 {
-    display: flex;
+    text-align: center;
     padding: 2.4rem 0 0.75rem;
-    align-items: center;
-    justify-content: center;
     font-weight: ${({ theme }): string => theme.fontBold};
     width: 100%;
     margin: 0 0 2.4rem;
-    font-size: 2.4rem;
+    font-size: 2rem;
     line-height: 1;
   }
+
+  ${media.mobile} {
+    > h1 {
+      line-height: 1.2;
+      margin-bottom: 1rem;
+    }
+  }
 `
 
 export const Home: React.FC = () => {
   return (
     <Wrapper>
-      <h1>Search Order ID / ETH Address / ENS Address</h1>
+      <h1>Search on Gnosis&nbsp;Protocol&nbsp;Explorer</h1>
```

Comment:
One more comment here. Can we remove the "&nbsp;" and use a simple space?

Reply:
We can, but i did it to keep "Gnosis Protocol Explorer " in one line on small mobile screens

Reply:
Let's try to decrease the font-size on mobile to fit the text in one line. Then we can remove the `&nbsp;`